src/index.ts

```diff
@@ -3,21 +3,12 @@
 * SPDX-License-Identifier: AGPL-3.0-or-later
 */
 
-import { translate, translatePlural } from '@nextcloud/l10n'
-import {
-	type Pinia,
+import type { Pinia } from 'pinia'
```

Comment:
`declare module 'vue/types/options.d.ts'` should not be needed in Vue 3, as far as I remember